### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'group'

### ISSUE THREAD

**Issue by alqasemy2020** [ading2210/poe-api#44]
Seems like the static variables in the HTML page are constantly changing
![image](https://user-images.githubusercontent.com/59507561/232259457-4be42265-fd0d-49b0-a252-d96182d8164f.png)
![image](https://user-images.githubusercontent.com/59507561/232259460-f889a0af-de3b-4c1e-85bd-048221c1cac6.png)

However, a minor change in the regex would solve this issue. https://github.com/ading2210/poe-api/blob/91a43003d25725fe958e58600325da2bb1216dbb/poe-api/src/poe.py#L83

  ```
script_regex = r'<script>if\(.+\)throw new Error;(.+)</script>'
script_text = re.search(script_regex, html).group(1)
key_regex = r'var .="([0-9a-f]+)",'
key_text = re.search(key_regex, script_text).group(1)
cipher_regex = r'.\[(\d+)\]=.\[(\d+)\]'
cipher_pairs = re.findall(cipher_regex, script_text)
```

**Comment by TheLime1** [comment]
#43 
i upgraded the code now then i got another error XD

**Comment by TheLime1** [comment]
PR please

**Comment by Tescalus** [comment]
Experiencing the same issue 😞 . It seems like a long-term solution is needed.

**Comment by ading2210** [comment]
> Experiencing the same issue 😞 . It seems like a long-term solution is needed.
@Tescalus 

Poe can always change their website subtly and it would break this library, there really isn't a possible way to prevent that other than trying to quickly find a bypass whenever they patch it.

**Comment by ading2210** [comment]
This has been fixed in 0.2.12.
